main.go

```diff
@@ -882,6 +882,7 @@ func ReleaseRepositoryRulesetArgs(repository *github.Repository, extraStatusChec
 			},
 			RequiredStatusChecks: &github.RepositoryRulesetRulesRequiredStatusChecksArgs{
 				RequiredChecks:                   requiredChecks,
+				DoNotEnforceOnCreate: pulumi.Bool(true),
```

Comment:
This is the actual change. The update of the github sdk was required for this field to be available in the first place and I updated all packages while I was at it with `go mod tidy`.